contracts/cw4-group-sorted/src/contract.rs

```diff
@@ -0,0 +1,189 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    attr, to_binary, Binary, Deps, DepsMut, Env, MessageInfo, Response, StdResult, SubMsg,
+};
+use cw2::set_contract_version;
+use cw4::{MemberChangedHookMsg, MemberResponse, TotalWeightResponse};
+
+use crate::error::ContractError;
+use crate::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+use crate::query::{Member, MemberListResponse};
+use crate::state::{
+    initialize_members, list_members_sorted, remove_member, update_member, ADMIN, HOOKS, MEMBERS,
+    TOTAL,
+};
+
+// version info for migration info
+const CONTRACT_NAME: &str = "crates.io:string-manager";
```

Comment:
Maybe a bit pedantic but do we want to call it string-manager over something like cw4-group-sorted?

Reply:
```suggestion
const CONTRACT_NAME: &str = "crates.io:cw4-group-sorted";
```